### ISSUE TITLE
Albums and Tracks Appear to Queue Differently

### ISSUE THREAD

**Issue by DavidLeigh** [gnumdk/lollypop#1296]
Ubuntu 16.04 dual boot with Windows 7 Professional 
Ubuntu updated today.
Lollypop version current with today (v0.9.304)

Again, thank you Cedric for a wonderful player.  I became infatuated with it from the outset.

I normally queue a 77-second track of silence before I queue an album.  (This allows my computer screen to shut off before the music begins ensuring no audible interruption.)  I could not do this in Lollypop by queuing the track and then queueing the album from the cover-art icon.  I had to queue the track, open the album, and queue all of the tracks from the album.  

The matter is a silly "three click difference" about which I would never complain, but since you are developing and improving Lollypop, I wished to share.  Also, if it helps, the reverse was impossible -- I could not queue an album from its cover-art icon and then queue a track.  I did not test any further, but would be happy to if  I missed something

I do not have Internet connection at home so their may be delays in my replying here.

Thanks for a wonderfully divergent, attractive, and innovative music player!

Dave....

**Comment by gnumdk** [comment]
Right click on album => queue?

**Comment by d9h20f** [comment]
What do you mean, 'no audible interruption'? Does locking your screen interrupt audio playback?

Also, did you know you can immediately lock your screen by using Super+L, or by opening the menu on the top right and clicking the 'close' icon?

**Comment by DavidLeigh** [comment]
To gnumdk... Yes.  If I queue a track I cannot then queue an album via the album art icon's + sign.  I can click the album art to open the track listing, click the > sign to open the sub-menu, and then click the + sign to load the tracks with the first track.  But NOT from the album art itself.  The reverse is also true.  If I queue an album from the album art icon's + sign, I cannot then queue a track.

**Comment by DavidLeigh** [comment]
To d920f...  When my screen shuts-off from the timer (in Ubuntu or Windows) there is a tiny disruption in the audio stream.  I cannot tell if it is an addition to or a subtraction from the sound, but is definitely a tiny disruption.  Queuing a silent track gives me time to close-up my computer, settle into my headphones and pillow, and have that little blip happen when it does not matter.  

I did not know that I could close the screen manually by either technique.  Thank you for sharing.

**Comment by gnumdk** [comment]
Right click on album => queue?

**Comment by gnumdk** [comment]
https://youtu.be/pB2A-WAgLKs

**Comment by DavidLeigh** [comment]
Thank you Cedric.  I understand how to do it.; just like the video. What I am sharing is that queuing a track or tracks in that manner cannot be then followed by queuing an album from the rollover + sign on the album art icon.  Or vice versa.  If I am unclear, I can try to make a video and push it to Google Cloud.  Would you like me to do that?

**Comment by gnumdk** [comment]
No, queue is an alternative-tmp playlist.

**Comment by gnumdk** [comment]
BTW, I plan to rework all the playlist behaviour in 2018 allowing user to add tracks in main playlist

**Comment by DavidLeigh** [comment]
Okay, thank you.  Lollypop is a wonderful player that had my affection as soon as I learned it.  Thank you for helping me with the other issue. I hope I was of some benefit to you regarding queue/playlist management.  Since you are reworking the queue/playlist function next year, I may make that clarifying video anyway.  Even if you do not want it, I will feel like I have offered something of value to your very valuable Lollypop.  Thanks again, Dave...
